### ISSUE TITLE
profile.py return code: 1

### ISSUE THREAD

**Issue by gdarm** [munkireport/munkireport-php#237]
Just tried new version of MR (2.4.0.1042) on 10.10.2 and 10.10.3 client.

But on other clients I got this:
```preflight stderr: WARNING: Munkireport: profile.py return code: 1```

Okay so far. MR runs fine for the rest of scripts.
Then I took a deeper look into the return code and got this:

```
Munkireport: profile.py Error: Traceback (most recent call last):
  File "/usr/local/munki/preflight.d/profile.py", line 73, in <module>
    txtfile.write(result)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xfc' in position 11950: ordinal not in range(128)
```

Then I run this command to see whats wrong with the profile:
```system_profiler -xml SPConfigurationProfileDataType```

But I can't see it...

I'm using Profile Manager on OS X 10.10.2 with Server App 4.0.3. (Upgrade to 4.1 planed for Monday. ;) )

Maybe this is because of some german special characters like ä or ö or similar...
But don't know to handle this... :/

**Comment by bochoven** [comment]
I added a fix here 832b58e22e5e763e154f14728c8623b603bf903f

**Comment by gdarm** [comment]
Just tested, worked fine now. :)
